docs/deployment/heroku.md

```diff
@@ -8,6 +8,27 @@ The only thing we'll need for this guide is a working Phoenix application. For t
 
 Our main goal for this guide is to get a Phoenix application running on Heroku.
 
+### Limitations
+
+Heroku is a great platform overall, but for Elixir apps, it comes with a number of limitations you should be aware of. 
```

Comment:
Oh yes, absolutely. I'll make those changes. I didn't mean for it to sound like a blocker. 

My goal in building Gigalixir was actually to help increase Elixir adoption since I fell in love with the language and ecosystem. I noticed that deployment seemed to be the most common pain point for people new to Elixir so I thought making deployment easier would make it more likely for people to choose Elixir. If you agree with that, do you think including a reference to Gigalixir somewhere in the docs would be mutually beneficial to Elixir, Gigalixir, and the developers?